README.md

```diff
@@ -42,3 +42,6 @@ We defined a Makefile with many useful commands for local development. These com
 ## Deployment
 ### QA
 * Deploys to https://tupress-qa.k8s.temple.edu
+
+### Production
+* To deploy the app to the prod cluster, add a new release via gitlab. The release image will be tagged with "prod" and with the release tag you use in the release.
```

Comment:
"The release image will be tagged with "prod" and with the release tag you use in the release."

Can this be changed to be more clear? 

The release image should be tagged with "prod" and the release tag normally used for releases, e.g. "prod, 1.3.3".

Reply:
@cdoyle-temple by "should be tagged with 'prod'" is that something you believe you are doing manually.  What I'm trying to say is that gitlab will automatically assign a prod tag to the release.

Reply:
Hence my confusion, Yes:

`The release image will automatically be tagged with "prod". The release tag normally used for releases should continue to be added manually, e.g. "1.3.3".`